docs/src/modules/components/AppFrame.js

```diff
@@ -68,14 +68,14 @@ const languages = [
     code: 'es',
     text: '🇪🇸 Español',
   },
-  // {
-  //   code: 'de',
-  //   text: '🇩🇪 Deutsch',
-  // },
-  // {
-  //   code: 'ja',
-  //   text: '🇯🇵 日本語',
-  // },
+  {
+    code: 'de',
+    text: '🇩🇪 Deutsch',
+  },
+  {
+    code: 'ja',
```

Comment:
Is it intended that this change also unlocks new translations? If so we should make sure that this doesn't get lost in the changelog.

Reply:
@eps1lon It doesn't unlock new translations. It's fixing an issue where people can see "locked" translation on https://material-ui.com/.